misc/config_tools/static_allocators/guest_flags.py

```diff
@@ -22,7 +22,7 @@ class GuestFlagPolicy(namedtuple("GuestFlagPolycy", ["condition", "guest_flag"])
     GuestFlagPolicy(".//nested_virtualization_support = 'y'", "GUEST_FLAG_NVMX_ENABLED"),
     GuestFlagPolicy(".//security_vm = 'y'", "GUEST_FLAG_SECURITY_VM"),
     GuestFlagPolicy(".//vm_type = 'RTVM'", "GUEST_FLAG_RT"),
-    GuestFlagPolicy(".//vm_type = 'RTVM' and .//load_order = 'PRE_LAUNCHED_VM'", "GUEST_FLAG_PMU_PASSTHROUGH"),
+    GuestFlagPolicy(".//vm_type = 'RTVM' and .//load_order = 'PRE_LAUNCHED_VM' and //hv/BUILD_TYPE= 'debug'", "GUEST_FLAG_PMU_PASSTHROUGH"),
```

Comment:
For the guest flag `GUEST_FLAG_PMU_PASSTHROUGH`, another point to take care of is excluding it from `DM_OWNED_GUEST_FLAG_MASK` for a release build. Otherwise, a DM may still be able to turn that on even a release-mode hypervisor is deployed.

Reply:
will add it later